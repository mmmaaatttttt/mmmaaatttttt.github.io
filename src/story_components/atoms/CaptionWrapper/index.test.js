import React from "react";
import { render } from "@testing-library/react";
import CaptionWrapper from "./";

it("renders without crashing", () => {
  render(<CaptionWrapper />);
});

it("matches snapshot with props and children", () => {
  const { asFragment } = render(
    <CaptionWrapper caption="I'm a caption!">
      <p>I'm a p tag with a caption!</p>
    </CaptionWrapper>
  );
  expect(asFragment()).toMatchSnapshot();
});
