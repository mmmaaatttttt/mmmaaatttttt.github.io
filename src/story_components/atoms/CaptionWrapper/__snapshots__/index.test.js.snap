// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with props and children 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa dQSUxY"
  >
    <p>
      I'm a p tag with a caption!
    </p>
    <p
      class="sc-bwzfXH kBfqmQ"
    >
      I'm a caption!
    </p>
  </div>
</DocumentFragment>
`;
